Sources/AccessibilitySnapshot/Core/Swift/Classes/AccessibilityHierarchyParser.swift

```diff
@@ -601,9 +601,14 @@ private extension NSObject {
                     )
                 )
             }
+
+            let explicitlyOrdered = accessibilityElements.contains {
+                "\(type(of: $0))" != "UILayoutContainerView"
```

Comment:
I don't love this fix but from my very small set of tests it works. Next week I'm going to check against our tests at work to see if anything breaks with this change.

Reply:
I've checked our tests at work and see no regression from this change.